app/views/pages/_show.slim

```diff
@@ -6,6 +6,7 @@
 = @rendered
 = render 'pages/personalization', data: @data
 = render 'pages/campaigner_overlay', page: @page
+= render 'pages/ak_credentials', page: @page
```

Comment:
does this expose `window.ak_credentials` on every page even if they're not a logged in user?